stdlib/bigarray.mli

```diff
@@ -210,7 +210,7 @@ val kind_size_in_bytes : ('a, 'b) kind -> int
 (** {1 Array layouts} *)
 
 type c_layout = C_layout_typ (**)
-(** See {!Bigarray.fortran_layout}.*)
+(** See {!Bigarray.type-fortran_layout}.*)
```

Comment:
This syntax is not supported by ocamldoc, the ocamldoc version is `{!type:Bigarray.type-fortran_layout}`.